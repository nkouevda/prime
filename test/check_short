#!/usr/bin/env bash

# Nikita Kouevda
# 2014/12/19

bin/prime -s 1 2 3 4 17 18 19
