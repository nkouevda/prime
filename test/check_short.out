4 of 7 (57.142857%)
